### ISSUE TITLE
Add an Unlikely Changes section to the Examples

### ISSUE THREAD

**Issue by niazim3** [smiths/caseStudies#100]
As mentioned in issue #47, a section for Unlikely Changes can be added to the examples if deemed fit. Please check them off as the task is looked into.

- [ ] GlassBR : @samm82 
- [ ] GamePhysics : @Mornix @deviprasad135 
- [ ] SWHS : @elwazana 
- [ ] NoPCM : @elwazana

Upon getting the changes in the manual versions approved by @smiths, please propagate the changes to Drasil. A format similar to JacquesCarette/Drasil#685 can be followed.

**Comment by samm82** [comment]
@smiths Can I propagate fd161a2 to Drasil?

**Comment by smiths** [comment]
@samm82, after you make the changes I noted in the above cited commit, yes, go ahead and propagate to Drasil.

**Comment by samm82** [comment]
@smiths Is 521248e better?

**Comment by smiths** [comment]
Sounds good.

**Comment by samm82** [comment]
Should we be adding Unlikely Change to this table...

![tableAbbAndAcro](https://user-images.githubusercontent.com/35857611/41679285-a7de4d0a-749b-11e8-8950-ee4854293808.png)

...and this list?

![assumpList](https://user-images.githubusercontent.com/35857611/41679341-ea7c925c-749b-11e8-8622-caf65e3eb16e.png)

**Comment by smiths** [comment]
Yes and yes.  I really like the idea of adding traceability to unlikely changes to the assumptions.  I hadn't thought of that previously, but it is great information to capture explicitly.  @szymczdm and @JacquesCarette will likely see a connection between Ernie Mileta's simplifications versus limitations.  I believe we could translate our assumptions to Ernie's categories if we added information about likely versus unlikely.

**Comment by niazim3** [comment]
@deviprasad135 @Mornix Sorry, I should've mentioned earlier to please coordinate with each other on who's making changes for this issue. I didn't know who to assign the issue since GamePhysics is assigned to you both.

**Comment by elwazana** [comment]
@smiths Hello Dr. Smith, can I get approval for the swhs and noPCM examples Unlikely changes so I can start to propagate them into the Drasil side of things?

**Comment by smiths** [comment]
@elwazana, please see the comments on your commit.  Once you make the changes, you can proceed to Drasil.  In the future, if you would like a comment on your commit, please ask in a comment.  I don't always notice commits, but a comment that sends an at (@) in my direction is always noticed.  😄

**Comment by samm82** [comment]
As noticed in JacquesCarette/Drasil#884, before this issue gets closed, the Unlikely Changes added should be added to the Traceability Matrices (and Graphs?) of caseStudies and Drasil

**Comment by smiths** [comment]
No, we don't have to add the unlikely changes to the traceability matrices and graphs.  Since they are unlikely changes, we don't have to plan for them occurring.  Moreover, an unlikely change will probably trace to multiple places in the documentation because the unlikely change will have far reaching consequences.  The traceability graph doesn't hold very useful information if it is almost fully populated.

The only reason to add the unlikely changes to the traceability matrix would be to test whether it is really necessary to consider them as unlikely changes.  If the traceability matrix is sparse, the developer may decide to accommodate the unlikely change, since it is easy to accommodate.  I don't think this is good enough of a reason to go through this extra work.  😄 

If the unlikely changes section has been added to all of the examples, then we can close this issue.

**Comment by samm82** [comment]
@smiths We're waiting on @elwazana for NoPCM and the merge of @Mornix's #102

**Comment by smiths** [comment]
Okay, I can see that now from looking at the checklist at the top of the issue.  Good job staying on top of things!
